### PR TITLE
Documentation for prompt()

### PR DESCRIPTION
**Pull request by hcarneiro**

### CODE REVIEW

File: docs/API/fliplet-core.md [hunk 1]
```diff
@@ -674,6 +674,26 @@ Fliplet.Navigate.confirm(options)
...
+```js
+var options = {
+  title: 'Foo',
+  message: 'Bar',
+  labels: ['Send','Cancel'], // Native only (defaults to [OK,Cancel])
+  default: 'Foo Bar'
+};
+Fliplet.Navigate.prompt(options)
+  .then(function(input) {
+    if (!input) {
```
Comment (tonytlwu) [inline docs/API/fliplet-core.md]: Should've used:

```js
if (input === null) {
  return console.log('Cancelled');
}

if (input === '') {
  return console.log('Left empty');
}
```

Comment (hcarneiro) [reply docs/API/fliplet-core.md]: @tonytlwu Just tested on Staging and pressing Cancel or pressing OK with the field empty gets resolved with false.

Only pressing OK with data in the field resolves with the data.

Comment (hcarneiro) [reply docs/API/fliplet-core.md]: @tonytlwu But I see what you mean, you wanted to be able to distinguish between the two.

Comment (hcarneiro) [reply docs/API/fliplet-core.md]: The code wasn't built to handle that but we can change.
https://github.com/Fliplet/fliplet-api/blob/master/public/assets/fliplet-core/1.0/navigate.js#L720-L742

**Pull request merged**